webapp/views.py

```diff
@@ -407,7 +407,7 @@ def get_context_data(self, **kwargs):
 
         try:
             with open(mirrors_path) as rss:
-                mirrors = parse(rss).entries
+                mirrors = parse(rss.read()).entries
```

Comment:
Actually, @tbille added this in his PR, which is already merged to the django2 branch. If you rebase, I think you'll find this is already there.